libs/sdk-ui-kit/src/LoadingMask/LoadingMask.tsx

```diff
@@ -2,20 +2,26 @@
 import React, { useMemo, CSSProperties } from "react";
 import cx from "classnames";
 
+/**
+ * @internal
+ */
+export type SpinnerSize = "large" | "small";
+
 /**
  * @internal
  */
 export interface ILoadingMaskProps {
     className?: string;
     height?: CSSProperties["height"];
     width?: CSSProperties["width"];
+    size?: SpinnerSize;
 }
 
 /**
  * @internal
  */
 export const LoadingMask: React.FC<ILoadingMaskProps> = (props) => {
-    const { className, height, width } = props;
+    const { className, height, width, size } = props;
```

Comment:
The size used to default to "large" now it does not. Is that ok?

Reply:
line 34 const spinnerSize: SpinnerSize = size ? size : "large";